nmrfx-processor/src/main/java/org/nmrfx/processor/datasets/vendor/NMRDataUtil.java

```diff
@@ -177,8 +177,12 @@ public static String isFIDDir(String fpath) {
             return bpath.toString();
         } else if (VarianData.findFID(bpath)) {
             return bpath.toString();
+        } else if (RS2DData.findFID(bpath)) {
+            return bpath.toString();
         } else if (JCAMPData.findFID(bpath)) {
             return bpath.toString();
+        } else if (JeolDelta.findFID(bpath)) {
+            return bpath.toString();
```

Comment:
Tested with the Jeol jdf files from the testfids we have, and they also couldn't drag and drop so added this here. Didn't add anything for JEOL in the isDatasetFile method, as it looks we don't load Jeol files as datasets anywhere else.